packages/cadl-ranch-specs/http/azure/resource-manager/models/common-types/managed-identity/mockapi.ts

```diff
@@ -0,0 +1,135 @@
+import { passOnSuccess, mockapi, json, ValidationError } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+const SUBSCRIPTION_ID_EXPECTED = "00000000-0000-0000-0000-000000000000";
+const RESOURCE_GROUP_EXPECTED = "test-rg";
+const validSystemAssignedManagedIdentityResource = {
+  id: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  location: "eastus",
+  identity: {
+    type: "SystemAssigned",
+    principalId: "00000000-0000-0000-0000-000000000000",
```

Comment:
Can't we just reference the SUBSCRIPTION_ID_EXPECTED above instead of putting 00000000-0000-0000-0000-000000000000 here?

Reply:
Fixed in the new version.